src/main/resources/openapi/eusage-reports-1.0.yaml

```diff
@@ -20,6 +20,18 @@ paths:
         description: limit titles associated with usage provider (UUID)
         schema:
           type: string
+      - in: query
```

Comment:
Might be nice to pull this out into whatever OpenAPI's analogue of a RAML trait is?

Reply:
Yes.. This becomes more and more apparent that this would be good to do.

Reply:
Now done.